src/index.ts

```diff
@@ -782,8 +784,9 @@ export default class IlpPluginAsymServer extends MiniAccountsPlugin {
     const newDropBalance = util.xrpToDrops(this.baseToXrp(newBalance))
     const encodedClaim = util.encodeClaim(newDropBalance.toString(), clientChannel)
     const keyPairSeed = util.hmac(this._secret, CHANNEL_KEYS + account.getAccount())
-    const keyPair = nacl.sign.keyPair.fromSeed(keyPairSeed)
-    const signature = nacl.sign.detached(encodedClaim, keyPair.secretKey)
+    const keyPair = sodium.Key.Sign.fromSeed(keyPairSeed)
+    const signer = new sodium.Sign(keyPair)
```

Comment:
Not sure how intensive it is, but would it be worth caching the key pair or signer instance on the account itself so it doesn't need to be re-generated each claim?

Reply:
fixed